frontend/src/constants/cartConstants.js

```diff
@@ -0,0 +1,2 @@
+export const CART_ADD_ITEM = 'CART_ADD_ITEM';
```

Comment:
Codacy has a fix for the issue: [Replace `'CART_ADD_ITEM'` with `"CART_ADD_ITEM"`](https://app.codacy.com/gh/Durotemy/e-commerce/pullRequest?prid=9481957)
```suggestion
export const CART_ADD_ITEM = "CART_ADD_ITEM";
```